concepts/azimuth/wallet.udon

```diff
@@ -21,40 +21,55 @@ are also derived from the wallet.
 Urbit HD wallets are composed of the following items, which are each assigned to
 their own individual Ethereum key-pairs.
 
-#### Master Ticket
+## Master Ticket
 
 Think of your master ticket like a very high-value password. The master ticket
 is the secret code from which all of your other keys are derived. Technically,
 your master ticket is seed entropy. You should never share it with anyone, and
 store it very securely. This ticket can derive all of your other keys: your
 ownership key and all of the related proxies.
 
-#### Ownership Address
+## Ownership Address
 
 An ownership address has all rights over the assets deeded to it. These rights
 are on-chain actions described and implemented in the Ecliptic, Azimuth's suite
 of governing smart-contracts.
 
-#### Proxies
+## Proxies
 
 Proxy addresses allow you to execute non-ownership related actions like spawning
 child points, voting, and setting networking keys without jeopardizing the keys
 you've designated with ownership rights. Setting proxy rights is optional, but
 it is recommended for on-chain actions you will execute more frequently.
 
-- *Management Proxy*
+### Management Proxy
 
-  Can configure or set Arvo networking keys and conduct sponsorship related
-  operations.
+Can set Arvo networking keys and conduct sponsorship-related operations.
 
-- *Voting Proxy*
+#### Networking Keys
 
-  Galaxies only. Galaxies are the part of the galactic senate, and this means
-  they can cast votes on new proposals including changes to the Ecliptic.
+Networking keys -- encryption and authentication key-pairs -- are used
```

Comment:
I believe "key pairs" would be more correct.

Reply:
Not sure I agree here. It's used elsewhere as a compound noun (keypair), and the hyphen similarly reduces ambiguity. "Pair of keys" is the indivisible concept here. Compare "old-key pair" to "old key-pair." If we write it as "old key pair," it's unclear which one of these ideas is the one we want to convey.

Reply:
Eh, fair enough. I'd probably go with "keypairs" then for consistency, but do what you think is best here.

Reply:
Surprised to see "key pair" is possibly more common, but "keypair" is also accepted https://en.wiktionary.org/wiki/keypair